packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -118,7 +117,11 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
                     this.hostPolicy.enableSingleRequestForShareLinkWithCreate,
             },
             async (event) => {
-                odspResolvedUrl = await createNewFluidFile(
+                // We can delay load this module as this path will not be executed in load flows and create flow
+                // while only happens once in lifetime of a document happens in the background after creation of
+                // detached container.
+                const module = await import("./internalModule");
```

Comment:
I'd suggest adding some telemetry and understanding it better. You are right that create flow will be used less often, but there is good chance this is the first Fluid action in given session, i.e. we did not load this module yet when we create a file. And if so - we might want to isolate create-file code into its own bundle, such that it is smaller, and thus help with create-flow perf, but also (indirectly) helps other flows by not loading this code.

Reply:
Done.